src/plugins/auto/plugin.cpp

```diff
@@ -95,7 +95,8 @@ std::vector<DeviceInformation> MultiDeviceInferencePlugin::ParseMetaDevices(cons
 
     auto getDeviceConfig = [&] (const DeviceName & deviceWithID) {
         DeviceIDParser deviceParser(deviceWithID);
-        std::string deviceName = deviceParser.getDeviceName();
+        std::string deviceName =
```

Comment:
How about if we have mock.0 and mock.1 devices? Should not use GPU here, consider to update the devcieParser, or add a getDeviceNameWithId()